tests/test_latentdistributiontest.py

```diff
@@ -14,47 +14,85 @@ class TestLatentDistributionTest(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         np.random.seed(123456)
+        cls.tests = ["dcorr", "mgc"]
```

Comment:
reminder to add a test with workers > 1

Reply:
reminder to add a test with a callable distance func as input (unless I missed this)